pkg/test/environment.go

```diff
@@ -76,7 +76,7 @@ type Environment struct {
 	InstanceProvider             instance.Provider
 	PricingProvider              *pricing.Provider
 	KubernetesVersionProvider    kubernetesversion.KubernetesVersionProvider
```

Comment:
Note that other caches are above (LoadBalancerCache, etc)

Reply:
Yea, I leaned away from that patterning, but we discussed offline, and decided that IoC seems solid for better testing options.